Remove Intel NDA from qsv11 plugin

AidanDelaney opened the pull request:
Intel committed an NDA disclaimer on each source file.  The stated
intention was that the NDA "added to OBS doesn't apply to open source
code once it's been accepted by the community. You can remove it for
your modifications".  This quote is from an email chain involving
Intel's legal team and developers.  The NDA in the source files
mistakenly triggers source code scanners that look for license
violations.  I have removed the comments that contain the NDA.

<!--- Please fill out the following template, which will help other contributors review your Pull Request. -->

<!--- Make sure you’ve read the contribution guidelines here: https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst -->

### Description
I have removed the Intel NDA from the QSV11 plugin.   The removal follows an email conversation between me, the original author (Seung-Woo) and the current person at Intel responsible for their contributions to OBS (Christine).  I can forward the email chain to any reviewer.

### Motivation and Context

When running a licence violation scanner over the OBS source code, the scanner shows that OBS code to be not open-source.  Such scanners are routinely deployed by companies before software is used in-house.  As such, removing the Intel NDA will allow more users to use OBS.

Ordanarily, it is not legal to remove a copyright header from a source file.  Intel have confirmed, in an email chain, that the NDA can be removed.  I can forward the email chain to any interested reviewer.

This PR does not fix an open issue as the substance of the changes does not fit within any of the new-issue templates.

### How Has This Been Tested?
I have removed comments from the top of source files in the QSV11 plugin.  The removal of comments will have no impact on the compiled binary.

### Types of changes
<!--- What types of changes does your PR introduce? Uncomment all that apply -->
<!--- - Bug fix (non-breaking change which fixes an issue) -->
<!--- - New feature (non-breaking change which adds functionality) -->
<!--- - Performance enhancement (non-breaking change which improves efficiency) -->
- Code cleanup (non-breaking change which makes code smaller or more readable)
<!--- - Breaking change (fix or feature that would cause existing functionality to change) -->
<!--- - Documentation (a change to documentation pages) -->

### Checklist:
<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [X] My code has been run through [clang-format](https://github.com/obsproject/obs-studio/blob/master/.clang-format).
- [X] I have read the [**contributing** document](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst).
- [X] My code is not on the master branch.
- [X] The code has been tested.
- [X] All commit messages are properly formatted and commits squashed where appropriate.
- [X] I have included updates to all appropriate documentation.

WizardCM commented on plugins/obs-qsv11/device_directx9.cpp:
```diff
@@ -435,4 +425,4 @@ mfxStatus CD3D9Device::CreateVideoProcessors()
 
-#endif // #if defined(WIN32) || defined(WIN64)
\ No newline at end of file
+#endif // #if defined(WIN32) || defined(WIN64)
```
This change is unrelated.

AidanDelaney replied:
I've removed the addition of the `\n` a the end of this file.

WizardCM commented on plugins/obs-qsv11/device_directx9.h:
```diff
@@ -115,4 +105,4 @@ class CD3D9Device : public CHWDevice {
 
-#endif // #if defined( _WIN32 ) || defined ( _WIN64 )
\ No newline at end of file
+#endif // #if defined( _WIN32 ) || defined ( _WIN64 )
```
This change is unrelated.

AidanDelaney replied:
Similarly, I've removed the addition of the `\n` a the end of this file.

WizardCM commented on plugins/obs-qsv11/common_directx11.h:
```diff
@@ -37,4 +27,4 @@ void CleanupHWDevice();
 void ClearYUVSurfaceD3D(mfxMemId memId);
-void ClearRGBSurfaceD3D(mfxMemId memId);
\ No newline at end of file
+void ClearRGBSurfaceD3D(mfxMemId memId);
```
This change is unrelated.

AidanDelaney replied:
I've removed the addition of the `\n` a the end of this file.

WizardCM reviewed (changes requested):
Additionally, please prefix your commit with obs-qsv per the contribution guidelines.

Conan-Kudo commented:
@AidanDelaney It would probably be a good idea to have a specific license notice file somewhere instead that declares this. If you have an email chain for this, you could include a plaintext version with signoffs from the Intel folks in question. [Here's an example of this with Mesa](https://src.fedoraproject.org/rpms/mesa/blob/rawhide/f/Mesa-MLAA-License-Clarification-Email.txt).

AidanDelaney commented:
Thank you @WizardCM for the review. I have addressed all of your review points.  And thanks also to @Conan-Kudo for your review.  I have included the file `QSV11-License-Claification-Email.txt` which is similar to the Mesa example that you provided.  Other files in this contribution list thier licence as "This file is provided under a dual BSD/GPLv2 license." Do you feel it is necessary to also include this in each of the 10 files from which the NDA has been removed?

Conan-Kudo commented:
@AidanDelaney `QSV11-License-Claification-Email.txt` should be renamed to `QSV11-License-Clarification-Email.txt` (typo on the word `Clarification`. But otherwise that email clarification file is good.

The dual-licensed state doesn't really matter for OBS Studio, since the whole work is GPLv2+.

AidanDelaney commented:
Thank you.  I've fixed the misspelling of the filename.

AidanDelaney commented:
@WizardCM I've updated all the changes as per the OBS contribution guidelines.  Would you mind re-reviewing?

AidanDelaney commented:
Rebased to get the checks to re-run.
